views/components/form/submit_button.erb

```diff
@@ -1,14 +1,12 @@
 <% name ||= nil
-extra_class ||= nil %>
+extra_class ||= nil
+attributes ||= {} %>
 
 <%== render(
   "components/button",
   locals: {
     text: text,
     extra_class: extra_class,
-    attributes: {
-      "type" => "submit",
-      "name" => name
-    }
+    attributes: { "type" => "submit", "name" => name }.merge!(attributes)
```

Comment:
Not a bug in this commit, but I think it would be better to only add the `name` attribute if it is not `nil`.  I guess having `name=""` in the HTML isn't a violation of anything, and it doesn't result in an parameter with an empty name being submitted, but it would still be something I would like to avoid.

Reply:
That makes sense. I've added it to the button component because the submit_button uses it.
https://github.com/ubicloud/ubicloud/pull/2442/commits/77475e01db471725a22863d18c91e116abf23fed